### ISSUE TITLE
[ENHANCEMENT] "Find all in current..." - hide "Line xxxx:"

### ISSUE THREAD

**Issue by gawkla** [notepad-plus-plus/notepad-plus-plus#1907]
Please add possibility to avoid appending "Line xxx:" predecessor in Find results window. For me this information is useless, and strongly destruct text adjustment. I always spend a lot of time to remove this appended text.
![notepad](https://cloud.githubusercontent.com/assets/10233305/15650505/5fda1aa6-2679-11e6-899b-c9f6c11fac1d.png)

**Comment by cmeriaux** [comment]
Right click on one of the result line, copy and paste in a new buffer. You will get the raw line.

This is different than : click on the result buffer, select all, copy and paste in a new buffer.

What do you think of this tips ?

**Comment by gawkla** [comment]
Hi, thanks for this info. This helps a lot, but still isn't perfect. For me it would be better not to see "Line xxx:" text, or eventually have all lines adjusted to same width. I know that this is my point of view, my usage schema. I use notepad++ to filter and view tons of logs. I still see area to improve "Find result" behaviour, but posted here is most important to me :) I hope, more people will vote my idea.

P.S. Recurring "Find in result" (feature from last version) is great! I love this!

**Comment by gawkla** [comment]
Hi, refreshing this thread. It would be really nice to have this feature working like classic grep. "Find all in current..." should show one line only once. Currently if line has searched pattern more than once, all occurrences produce one line in output window. Sometimes this duplicate same lines in output.

**Comment by MetaChuh** [comment]
the behaviour of the find results is as designed and expected.

many thanks and best regards.

**Comment by sasumner** [comment]
Yea, the use case would only be valuable to a tiny fraction of users.
